examples/silex/web/index.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
```

Comment:
What does this app trace?

How about to add an example for tracing the web handlers?

Reply:
A root span is created for the main web request which is named after the requested path.

I'll add another inline example within one for the request handlers.